python/cudf/cudf/core/buffer.py

```diff
@@ -74,7 +74,7 @@ def __cuda_array_interface__(self):
             "data": (self.ptr, False),
             "shape": (self.size,),
             "strides": (1,),
-            "typestr": "|u1",
+            "typestr": "|i1",
```

Comment:
Do you know why this was changed from `"|u1"` to `"|i1"`?

Reply:
Seeing what happens if we switch back in PR ( https://github.com/rapidsai/cudf/pull/5231 ).

Reply:
IIRC this change was because we read the mask buffer as a column to do bitwise operations that are exposed via columns in libcudf and we don't support unsigned types yet.

I think specifically surrounding UDFs with nulls where we bitwise OR the masks after running the numba kernels.